src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -921,6 +954,32 @@ private JButton getCloseButton() {
 		return closeButton;
 	}
 
+	private JButton getPrintButton() {
+		printButton = new JButton(MessageBundle.getMessage("angal.common.print.btn"));
+		printButton.setMnemonic(MessageBundle.getMnemonic("angal.common.print.btn.key"));
+		printButton.setEnabled(true);
+
+		printButton.addActionListener(e -> {
+			if (inventory == null || inventory.getId() == 0 || inventory.getId() <= 0) {
```

Comment:
why is `inventory.getId() == 0`  in the OR list?  Isn't it covered by the condition `inventory.getId() <= 0` ?